src/pyOpenMS/pxds/BasicProteinInferenceAlgorithm.pxd

```diff
@@ -61,42 +64,42 @@ cdef extern from "<OpenMS/ANALYSIS/ID/BasicProteinInferenceAlgorithm.h>" namespa
         void run(libcpp_vector[ PeptideIdentification ] & pep_ids,
                  libcpp_vector[ ProteinIdentification ] & prot_ids) nogil except +
           # wrap-doc:
-          #   Performs basic aggregation-based inference per ProteinIdentification run. See class help.
-          #   
-          #   ----------
-          #   :param pep_ids: Vector of peptide identifications
-          #   :param prot_ids: Vector of protein identification runs. Scores will be overwritten and groups added.
-          #   :return: Writes its results into prot_ids
```

Comment:
Hey @jpfeuffer 
Have you tested it locally?
Because the parameters are picked up if we have the separator as in line 66.

Reply:
No, but what does the separator do? Except creating loads of warnings like
```
----------
docstring of pyopenms.IDConflictResolverAlgorithm.resolve:18: CRITICAL: Unexpected section title or transition.
```

Reply:
IF at all, then you need to have empty lines before and after. But I think they are useless in API docs
https://rest-sphinx-memo.readthedocs.io/en/latest/ReST.html#transition

Reply:
Not very sure, but I think it's needed for parameter highlighting in html. Ok, no worries I will test it in my local system.

Reply:
No they are not. If done correctly, they would create a horizontal rule aka Transition.
You are probably confusing them with different docstrings syntaxes.